packages/flutter/lib/src/widgets/automatic_keep_alive.dart

```diff
@@ -394,7 +392,7 @@ mixin AutomaticKeepAliveClientMixin<T extends StatefulWidget> on State<T> {
 
   @mustCallSuper
   @override
-  Widget build(BuildContext context) {
+  Widget? build(BuildContext context) {
```

Comment:
as discussed elsewhere, we should return _NullWidget() or something here

Reply:
Done